.github/workflows/ci.yml

```diff
@@ -10,32 +10,25 @@ jobs:
   ci:
     name: Run checks and tests over ${{matrix.otp}} and ${{matrix.os}}
     runs-on: ${{matrix.os}}
+    container:
+      image: erlang:${{ matrix.otp }}
 
     strategy:
       fail-fast: false
       matrix:
-        otp: ["25.3", "24.3"]
+        otp: ["26.0", "25.3", "24.3"]
```

Comment:
Nit: Why not just go with `26, 25, 24` no reason to target a specific version? :-)

Reply:
What, eg, `25` exactly mean? Would it install the latest 25.X version?

Reply:
Exactly :)